### DISCUSSION TITLE
Custom actions for alerts

### DISCUSSION THREAD

**Discussion by fffunke** [grafana/grafana#68478] [Alerting]
I've started experimenting with grafana and grafana on-call.
So far I did not see a possibility to add a custom action for an alert.

For example if I get an alert for a kubernetes pod being in `CrashLoopBackoff` I would like to get a notification and possibly create an issue in gitlab. 
The gitlab issue could be created with a webhook call. 

I would *NOT* like for grafana on-call to create a gitlab issue automatically (which would be possible), instead I would like to do that for alerts if I see the need for that. But only on a manual action (e.g. clicking a button)

Is there already an option to do that? Otherwise I think this feature would be really nice

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
